.gitignore

```diff
@@ -40,4 +40,9 @@ yarn-error.log*
 **/*.log
 package-lock.json
 **/*.bun
-idea/
\ No newline at end of file
+idea/
+
+# utility script to rebuild docker
+# image and restart the API after
+# changes to code
+/runner-script.sh
```

Comment:
There's the script mentioned in `.gitignore`, but no script is included in the PR.

Reply:
This was a script that I used to test the program locally, but I didn't want it included in the commit, so I added it to `.gitignore`. Can you please tell me what I should do in this case?

Reply:
In this case you could just remove the lines you added to `.gitinore` (from empty line 44 to line 48) and make sure you don't commit your script file.

Reply:
Right, fixed this.